lib/incoming_mail.rb

```diff
@@ -0,0 +1,110 @@
+module IncomingMail
```

Comment:
@afeld Agreed on `lib/tasks` -- I see that the distinction we having going her is that `app/models` is for AR classes and `lib` is for everything else. I think that is a fine distinction to run with, although not one I've seen in the past.

If we do continue to put all non-AR classes into `lib`, perhaps a good middle ground would be to have more sub-folders inside `lib` so we don't have a bunch of unrelated classes thrown into the same directory?